setup/.env

```diff
@@ -1,5 +1,5 @@
 # Hostname passed to Traefik
-ADDRESS=setup.mailu.io
+ADDRESS=test02.usrpro.io
```

Comment:
I don't think you meant to commit this. As a matter of fact, this file should be deleted, I've changed `docker-compose.yml` to allow for local running. All the deployment stuff has been moved to https://github.com/Mailu/infra.

If you want to run/deploy setup behind Treafik, I recommend taking files from that repository.